.gitignore

```diff
@@ -105,4 +105,8 @@ ENV/
 
 # mypy
 .mypy_cache/
+<<<<<<< HEAD
```

Comment:
It looks like you have an unresolved diff here. You should remove the lines with `<<<`, `====`, and `>>>>`.